peer.go

```diff
@@ -209,6 +211,29 @@ func (p *peer) isKnownInventory(invVect *wire.InvVect) bool {
 	return false
 }
 
+// UpdateLastBlockHeight updates the last known block for the peer. It is safe
+// for concurrent access.
+func (p *peer) UpdateLastBlockHeight(newHeight int32) {
+	p.StatsMtx.Lock()
+	defer p.StatsMtx.Unlock()
+
+	//if newHeight > p.lastBlock {
+	peerLog.Infof("Updating last block height of peer %v from %v to %v",
```

Comment:
Yeah it was a left over logging statement for tracing through execution. I've demoted it a `Tracef` log statement.